### ISSUE TITLE
Alfred 4 - workflow is unstable

### ISSUE THREAD

**Issue by avxkim** [DONSA/alfred-google-translate#25]
When i type a command `gt привет`:

![image](https://user-images.githubusercontent.com/5739191/61120731-6fa05c80-a4bf-11e9-874f-e513703207d3.png)

It doesn't translate, i have to delete the last letter and re-enter it again (sometimes it takes a few attempts):

![image](https://user-images.githubusercontent.com/5739191/61120784-89da3a80-a4bf-11e9-84f7-e1a01e4cd91e.png)

Is it google translate api changed or alfred?

Here's the logs:

```
[16:12:18.042] Google Translate[Script Filter] Queuing argument 'п'
[16:12:18.108] Google Translate[Script Filter] Queuing argument 'пр'
[16:12:18.278] Google Translate[Script Filter] Queuing argument 'при'
[16:12:18.346] Google Translate[Script Filter] Queuing argument 'прив'
[16:12:18.388] Google Translate[Script Filter] Script with argv '(null)' finished
[16:12:18.390] Google Translate[Script Filter] <?xml version="1.0" encoding="utf-8"?>
<items><item uid="en" arg="https://translate.google.com/#auto/en/%D0%BF|P" valid="yes"><title>P</title><subtitle>п (Automatic)</subtitle><icon>/Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/src/icons/en.png</icon></item></items>
[16:12:18.492] Google Translate[Script Filter] Queuing argument 'приве'
[16:12:18.582] Google Translate[Script Filter] Queuing argument 'привет'
[16:12:18.743] Google Translate[Script Filter] Script with argv '(null)' finished
[16:12:18.746] ERROR: Google Translate[Script Filter] Code 255: Fatal error: Uncaught ErrorException: Client error: `POST http://translate.google.com/translate_a/single?client=t&hl=en&dt=t&sl=auto&tl=en&ie=UTF-8&oe=UTF-8&multires=1&otf=0&pc=1&trs=1&ssel=0&tsel=0&kc=1&tk=543643.948453` resulted in a `400 Bad Request` response:
<!DOCTYPE html><html lang=en><meta charset=utf-8><meta name=viewport content="initial-scale=1, minimum-scale=1, width=de (truncated...)
 in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php:294
Stack trace:
#0 /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php(331): Stichoza\GoogleTranslate\TranslateClient->getResponse('\xD0\xBF\xD1\x80\xD0\xB8\xD0\xB2')
#1 /Users/alexanderkim/Library/Application Support/Alfred/Alfre in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php on line 294
[16:12:18.747] ERROR: Google Translate[Script Filter] JSON error: JSON text did not start with array or object and option to allow fragments not set. in JSON:
Fatal error: Uncaught ErrorException: Client error: `POST http://translate.google.com/translate_a/single?client=t&hl=en&dt=t&sl=auto&tl=en&ie=UTF-8&oe=UTF-8&multires=1&otf=0&pc=1&trs=1&ssel=0&tsel=0&kc=1&tk=543643.948453` resulted in a `400 Bad Request` response:
<!DOCTYPE html><html lang=en><meta charset=utf-8><meta name=viewport content="initial-scale=1, minimum-scale=1, width=de (truncated...)
 in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php:294
Stack trace:
#0 /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php(331): Stichoza\GoogleTranslate\TranslateClient->getResponse('\xD0\xBF\xD1\x80\xD0\xB8\xD0\xB2')
#1 /Users/alexanderkim/Library/Application Support/Alfred/Alfre in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php on line 294
[16:12:19.083] Google Translate[Script Filter] Script with argv '(null)' finished
[16:12:19.093] ERROR: Google Translate[Script Filter] Code 255: Fatal error: Uncaught ErrorException: Client error: `POST http://translate.google.com/translate_a/single?client=t&hl=en&dt=t&sl=auto&tl=en&ie=UTF-8&oe=UTF-8&multires=1&otf=0&pc=1&trs=1&ssel=0&tsel=0&kc=1&tk=251610.386468` resulted in a `400 Bad Request` response:
<!DOCTYPE html><html lang=en><meta charset=utf-8><meta name=viewport content="initial-scale=1, minimum-scale=1, width=de (truncated...)
 in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php:294
Stack trace:
#0 /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php(331): Stichoza\GoogleTranslate\TranslateClient->getResponse('\xD0\xBF\xD1\x80\xD0\xB8\xD0\xB2\xD0\xB5\xD1\x82')
#1 /Users/alexanderkim/Library/Application Supp in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php on line 294
[16:12:19.095] ERROR: Google Translate[Script Filter] JSON error: JSON text did not start with array or object and option to allow fragments not set. in JSON:
Fatal error: Uncaught ErrorException: Client error: `POST http://translate.google.com/translate_a/single?client=t&hl=en&dt=t&sl=auto&tl=en&ie=UTF-8&oe=UTF-8&multires=1&otf=0&pc=1&trs=1&ssel=0&tsel=0&kc=1&tk=251610.386468` resulted in a `400 Bad Request` response:
<!DOCTYPE html><html lang=en><meta charset=utf-8><meta name=viewport content="initial-scale=1, minimum-scale=1, width=de (truncated...)
 in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php:294
Stack trace:
#0 /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php(331): Stichoza\GoogleTranslate\TranslateClient->getResponse('\xD0\xBF\xD1\x80\xD0\xB8\xD0\xB2\xD0\xB5\xD1\x82')
#1 /Users/alexanderkim/Library/Application Supp in /Users/alexanderkim/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.D58C2EF6-B5A6-485A-936D-AE98E3ACA875/vendor/stichoza/google-translate-php/src/Stichoza/GoogleTranslate/TranslateClient.php on line 294
```

**Comment by DONSA** [comment]
Yes, their API changed. I still didn't have time to look into it.
PR's are welcome 😄

**Comment by DONSA** [comment]
Fixed on ee25ed8. Please update it.
